### PR TITLE
feature/224 - Multilingual Support

### PR DESCRIPTION
**Pull request by gangjun06**
#224 
### Golang
- Library: system/core/i18n/i18n.go
### NextJS
- Used Library: https://www.npmjs.com/package/next-i18next
## TODO
- [x] Multilang support on System b0776d51d48f3864edb6cec17228d1470411d8bc, 59a7a5befae9d74ebe5e16effbae1854ceae9a3c
  - [x] Translate Into Korean 94a9ef3cbca060e605927f4b942a6d29f831d2d0, 5cefee6263b81b613cd37c86e6ab9a35008a2485
- [x] Add i18n to youtube-monitor 22d74c5efc7ea28143b6610580f0f62af9ee7f45
  - [x] Translate Into Korean 9e30f6ba08fda343c1975830d28affdd8f7ae6c3
- [x] Add i18n to 1000-minutes-simulator 720eee95aae83ff480c8813ceb0bcbc38c1528ee
  - [x] Translate UI Into Korean 764f913924a839d563c8459d3a39cdc100bf25d9

### CODE REVIEW

**Comment by gangjun06** [comment]
Translated 1000-minutes-simulator
ㅊan change the language by changing url ( example.com -> example.com/ko )
[CleanShot 2022-09-18 at 13 24 26](https://user-images.githubusercontent.com/50910815/190885659-5331cfeb-dd9e-441e-a245-069bf0866448.png)

**Comment by gangjun06** [comment]
Translated youtube-monitor
Can change the language by changing url ( example.com -> example.com/ko )
CleanShot 2022-09-18 at 14 46 55

**Comment by gangjun06** [comment]
Except for the i18n_test.go file, the actual functioning of the i18n module has not been tested.
I think it's better to check the functioning before merge.
Also I've checked, but some of the sentences may have been damaged while translating.

**Comment by sorarideblog** [comment]
@gangjun06 
Thank you for your cooperation!
I'll review codes and check if the program works properly, then merge this PR.

File: system/locales/ko.toml [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+"on" = "ON"
+"off" = "OFF"
+"sir" = "{0}님, "
+
+[command]
+"option-warn" = "{0}님, 옵션이 올바르게 설정되어 있는지 확인해주세요." # 0: Username
+"error" = "{0}님 오류가 발생하였습니다. 다시 시도해 주세요" # 0: Username
+"rp-earned" = "（{0} RP）" # 0: Earned
+"not-enter" = "{0}님, 아직 입장하지 않으셨습니다. 「{1}」로 입실해주세요!" # 0: Username, 1: InCommand
+"enter-only" = "{0}씨, 입실중에만 사용할 수 있는 커맨드입니다" # 0: Username  # TODO check
```
Comment (sorarideblog) [inline system/locales/ko.toml:13]: @gangjun06 
I added some sentences generated by google Translate in `ko.toml`. (see # TODO)
Can you check if these are natural, please?

Comment (gangjun06) [reply system/locales/ko.toml:13]: @sorarideblog 
3593f4ee3c2689a461c9b868c6d1f3ff12bed6d3
Yes, I checked.

**Review by sorarideblog — COMMENTED** [review]
Thanks!

**Pull request merged**